analytic_partner_hr_timesheet/models/res_partner.py

```diff
@@ -1,14 +1,15 @@
 # -*- coding: utf-8 -*-
-# (c) 2015 Antiun Ingeniería S.L. - Pedro M. Baeza
+# Copyright 2015 Antiun Ingeniería S.L. - Pedro M. Baeza
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
-from openerp import models, api, fields
+
+from odoo import api, fields, models
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     timesheets = fields.One2many(
```

Comment:
Rename it to `timesheet_ids` for following conventions. No need to put `oldname=...`